Doc/library/dis.rst

```diff
@@ -479,6 +479,20 @@ the original TOS1.
    Implements ``del TOS1[TOS]``.
 
 
+.. opcode:: BINARY_SLICE
+
+   Implements ``TOS = TOS2[TOS1:TOS]``.
```

Comment:
I think the preamble to this section in lines 449-456 needs to be updated because it only refers to TOS and TOS1.